docs/src/content/docs/components/input-select.mdx

```diff
@@ -73,10 +79,61 @@ Displays a dropdown field, allowing a user to select an option from a pre-define
 				placeholder: 'Choose one...',
 				initialValue: [{ label: 'Option 1', value: 'option1' }]
 			}
+		},
+		{
+			typeName: 'display.text',
+			props: {
+				label: 'Selected label: {{selectInput.label}}'
+			}
+		},
+		{
+			typeName: 'display.text',
+			props: {
+				label: 'Selected value: {{selectInput.value}}'
+			}
 		}
 	]}
 />
 
+## Runtime State
+
+The value in the runtime state is the string value of the select input, using dot notation to
+access the desired property.
+
+```ts title="lib/MyStack.ts" "'name'"
+Input.select({
+	id: 'selectInput',
+	options: [
+		{ label: 'Option 1', value: 'option1' },
+		{ label: 'Option 2', value: 'option2' }
+	],
+	label: 'Select an option'
+})
+```
+
+```json title="Runtime state (1st option selected)" frame="terminal" ""name""
```

Comment:
Why the double quotes here btw? `""`

Reply:
My bad, then it's okay. It's like that in order to highlight also the double quotes in the text. All good then 👍